third_party/ObjexxFCL/src/ObjexxFCL/vectorize.hh

```diff
@@ -23,16 +23,33 @@
 #define ASSUME(b)
 #define ASSUME_ALIGNED(p,b) p=(decltype(p))__builtin_assume_aligned(p,b)
 #define RESTRICT __restrict__
-#elif defined(_WIN32) && ( defined(_MSC_VER) || defined(__INTEL_COMPILER) )
+
+#elif defined(_WIN64) && ( defined(_MSC_VER) || defined(__INTEL_COMPILER) )
 #define ASSUME(b) __assume(b)
 #define ASSUME_ALIGNED(p,b) __assume_aligned(p,b)
 #define RESTRICT __restrict
+
+// Disable alignment hints for 32-bit windows builds
+#elif defined(_WIN32) && ( defined(_MSC_VER) || defined(__INTEL_COMPILER) )
+#define ASSUME(b) __assume(b)
+#define ASSUME_ALIGNED(p,b) 
+#define RESTRICT __restrict
+
 #else
 #define ASSUME(b)
 #define ASSUME_ALIGNED(p,b)
 #define RESTRICT
 #endif
 
+// Alignment for Compiler Options
+#if defined (__AVX2__)
```

Comment:
@amirroth Not a big deal but I meant for you to put that vectorize.hh in src/EnergyPlus as a layer on top of the ObjexxFCL vectorize.hh, keeping the EnergyPlus specifics out of ObjexxFCL.